parsers/EIA.py

```diff
@@ -70,54 +143,15 @@ def _fetch_production_or_consumption(zone_key, series_id, session=None,
     } for datapoint in raw_data['series'][0]['data']]
```

Comment:
It's `'eia.gov'` for the source I think.

Reply:
good catch, thanks, fixed.